.github/workflows/test.yaml

```
@@ -19,7 +19,7 @@ jobs:
           # When updating the minimum Python version here, also update the
           # `python_requires` from `setup.cfg`.
           # Run on latest minor release of each major python version.
-          python-version: ["3.6.7", 3.6, 3.7, 3.8, 3.9, pypy-3.6, pypy-3.7]
+          python-version: ["3.6.7", 3.6, 3.7, 3.8, 3.9, 3.10.0-alpha.5, pypy-3.6, pypy-3.7]
```

can we use just 3.10 and it will automatically pic last 3.10 version?

```suggestion
          python-version: ["3.6.7", 3.6, 3.7, 3.8, 3.9, 3.10, pypy-3.6, pypy-3.7]
```

---

I'd rather bump the pre-release version manually, so we don't subject Twisted CI to the whims of every single alpha breaking change